ironic/tests/unit/drivers/modules/drac/test_inspect.py

```diff
@@ -378,3 +387,81 @@ def test__get_pxe_dev_nics_with_empty_list(self, mock_get_drac_client):
                 mock_client, self.nics, self.node)
 
             self.assertEqual(expected_pxe_nic, pxe_dev_nics)
+
+
+class DracRedfishInspectionTestCase(db_base.DbTestCase):
+
+    def setUp(self):
+        super(DracRedfishInspectionTestCase, self).setUp()
+        self.config(enabled_hardware_types=['idrac'],
+                    enabled_power_interfaces=['idrac-redfish'],
```

Comment:
Instead of enabling the idrac-redfish version of the interfaces here, could we move this into BaseDracTest.setUp() and make this class inherit from BaseDracTest?

Reply:
Hi Chris,
Pradip here,
I have inherited test_utils.BaseDracTest without any issue.
  
But while moving 'idrac-redfish' to BaseDracTest.setUp() ,getting attached error msg.


<img width="960" alt="errMsg" src="https://user-images.githubusercontent.com/34384282/67575382-1a304880-f75a-11e9-9d75-d0741f0447ea.png">

Reply:
Ah, I see.  It looks like it's trying to run the redfish test case against the wsman inspect implementation.  Let's leave it as-is then.

Reply:
Okay